regulation-worker/internal/delete/api/api.go

```diff
@@ -76,6 +108,22 @@ func (api *APIManager) Delete(ctx context.Context, job model.Job, destConfig map
 		return model.JobStatusFailed
 	}
 
+	// Refresh Flow Start
+	if delAccountIdExists {
+		respParams := &handleRefreshFlowParams{
+			jobResponses: jobResp,
```

Comment:
previously, we were not using `jobResp`. Can you please confirm that transformer doesn't really sends response for each users in the payload that we sent?

Reply:
As discussed in the call, this is still kept for future needs.